modules/appeals_api/db/migrate/20210412175709_add_upload_submission_to_evidence_submissions.rb

```diff
@@ -0,0 +1,16 @@
+class AddUploadSubmissionToEvidenceSubmissions < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
```

Comment:
TIL. What does this do?

Reply:
strong migrations recommended it due to adding the upload_submission reference and adding the index, since that's a blocking operation.